packages/patterns/src/getOpaquePattern.ts

```diff
@@ -15,9 +15,13 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { Gradient, Pattern, PatternType } from './types';
+import {
+  Gradient,
+  Pattern,
+  PatternType,
+} from '@googleforcreators/types';
```

Comment:
@swissspidy, I am getting some very annoying conflicts between various formatters now. I have tried updating everything, but the issues persist.

Basically, when I auto-format this file in VScode using prettier (Opt+Shift+F), these three imports are inlined correct as:

```ts
import { Gradient, Pattern, PatternType } from '@googleforcreators/types';
```

However, when I save the file, some other formatter runs and re-formats the file as:

```ts
import {
  Gradient,
  Pattern,
  PatternType,
} from '@googleforcreators/types';
```

And this is just one of the minor problems caused by these unwanted reformats. Sometimes whole lines of imports (almost always types) are moved around in the file, causing imports to occur before `/** External dependencies */` causing other lint errors.

Do you have any idea what I could have configured wrong? Or do we have some new formatting rules for TS that isn't correctly synced with prettier and/or ESLint?

Reply:
Just checked and VSCode's built-in "format on save" setting was disabled. Enabling that causes both prettier format and the unknown formatter to run, prettier first and then the other one second.

It seems like some rogue watcher doing things wrong, but I definitely don't have anything running in my terminal. Closing all terminals and relaunching VSCode still has the problem persisting, so not really sure what could be going on here?

Reply:
Hah, I figured it out. I had installed a small package to do import sorting on request, but apparently it also did so on save, but it was an idiot at it, so...!